packages/nuxeo-web-ui-ftest/pages/ui/browser.js

```diff
@@ -371,4 +371,8 @@ export default class Browser extends BasePage {
     publishDialog.waitForVisible();
     return publishDialog;
   }
+
+  get comparePage() {
+    return this.el.element('nuxeo-page div[class="header"]');
```

Comment:
This helper is not specific enough and can return the wrong page. Consider adding the page element tagName to the query to make it more specific. Something like this:
```suggestion
    return this.el.element('nuxeo-diff-page div.header');
```